pkg/kubelet/lifecycle/predicate.go

```diff
@@ -98,7 +98,7 @@ func (w *predicateAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 	fit := len(reasons) == 0 && err == nil
 	if err != nil {
 		message := fmt.Sprintf("GeneralPredicates failed due to %v, which is unexpected.", err)
```

Comment:
```suggestion
```

Reply:
But the `msg`  variable is used [below](https://github.com/kubernetes/kubernetes/blob/9735ef75a62e19165b1a9c814b42789548729457/pkg/kubelet/lifecycle/predicate.go#L117)
```
		return PodAdmitResult{
			Admit:   fit,
			Reason:  "UnexpectedAdmissionError",
			Message: message,
		}
```